src/utils/formatStatusToKorea.ts

```diff
@@ -0,0 +1,9 @@
+import { MyReservationsStatus } from '@/types';
+
+export const formatStatusToKorea = (status: MyReservationsStatus) => {
+  if (status === 'pending') return '신청';
+  if (status === 'confirmed') return '승인';
+  if (status === 'declined') return '거절';
+  if (status === 'canceled') return '취소';
+  if (status === 'completed') return '종료';
```

Comment:
```suggestion
const name = {
  pending: '신청',
  confirmed: '승인',
  ...
}

return name[status];
```

이런식으로 객체화해서 리턴하는건 어떨까요!

Reply:
좋은 방법이네요! 덕분에 배워갑니다!